client/main.lua

```diff
@@ -1499,7 +1715,6 @@ function CreateBlips()
 
 	if Config.UseCarGarages and Config.UseCarBlips then
 		for k,v in pairs(Config.CarGarages) do
-			if not v.Hidden then
```

Comment:
removing this will make the config file invalid.

Reply:
Do not remove this line.

Reply:
This is an old commit po sir, I restored this line po sa commit #3